### PR TITLE
fixed_ockam identity show a prints unwanted % in the end

### PR DESCRIPTION
**Pull request by SamantaTarun**
#4707 
@tarunsamanta2k20 
## Current behavior
ockam identity show a prints unwanted % in the end

sanjo@Oleksandrs-MBP ockam % ockam identity create a
Default vault created: 5224c211
Identity created: P1d976ac05d6f1c50ff568a1404b1d5363714e76d67b4a285eefdd3c21408abd4
sanjo@Oleksandrs-MBP ockam % ockam identity show a
P1d976ac05d6f1c50ff568a1404b1d5363714e76d67b4a285eefdd3c21408abd4%
## Proposed changes
% shouldn't be there

## Checks

- [ ] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [ ] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [ ] I accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will mark the commit as verified.

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
@tarunsamanta2k20 Thank you for spending time on this, while we review and before we can merge please

- [x] Accept the Ockam [Contributor License Agreement](https://github.com/build-trust/ockam-contributors/blob/main/CLA.md) by adding your Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/blob/main/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam-contributors](https://github.com/build-trust/ockam-contributors) repository.

The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam-contributors/edit/main/CONTRIBUTORS.csv) file in the github web UI and create a PR, this will make the commit as verified.

File: implementations/rust/ockam/ockam_command/src/identity/show.rs [hunk 1]
```diff
@@ -51,14 +51,14 @@ fn run_impl(opts: CommandGlobalOpts, cmd: ShowCommand) -> crate::Result<()> {
     if cmd.full {
         let identity = state.config.change_history.export()?;
         if Some(EncodeFormat::Hex) == cmd.encoding {
-            print_output(identity, &opts.global_args.output_format)?;
+            println_output(identity, &opts.global_args.output_format)?;
```
Comment (caoakleyii) [inline implementations/rust/ockam/ockam_command/src/identity/show.rs:54]: Thank you for your contribution! 🎉

Small change is needed -

This `fn` is missing it's import: 

```rs
use crate::util::println_output;
```

This change also removes any usages of `print_output` and that associated `fn` can be removed too.

**Review by caoakleyii — CHANGES_REQUESTED** [review]

File: implementations/rust/ockam/ockam_command/src/identity/show.rs (hunk 1 above)
Comment (caoakleyii) [reply implementations/rust/ockam/ockam_command/src/identity/show.rs:54]: Thanks for the update.

I'm running the automated actions, I do believe the rust build will warn about `fn print_ouput` definition.

I can point you in that right direction if that is the case.

Comment (caoakleyii) [reply implementations/rust/ockam/ockam_command/src/identity/show.rs:54]: This is the error the build is running into:

https://github.com/build-trust/ockam/actions/runs/4760039036/jobs/8474497357?pr=4737#step:9:47


This is the `fn` in reference:

https://github.com/tarunsamanta2k20/ockam/blob/02323a58b856b9e6474642327d24738a68af38d4/implementations/rust/ockam/ockam_command/src/util/mod.rs#L337-L344

You can go ahead and delete that function, when you run `cargo build` in the root directory. The warning should no longer appear.

**Comment by caoakleyii** [comment]
One other thing to note is our[ commit linter](https://github.com/build-trust/ockam/actions/runs/4760039029/jobs/8474494361?pr=4737), we have a specific style of commit messages we expect.

You can read more about that [here.](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages)

Quick example of how this `fixed_ockam identity show a prints unwanted % in the end` message would fit into our expectations.

`fix(rust): fixes ockam identity show by removing the unwanted trailing percent symbol`

Thank you!

**Comment by davide-baldo** [comment]
Hi! I've rebased the commit to prepare it for merge! almost there...

**Review by polvorin — APPROVED** [review]

**Review by caoakleyii — APPROVED** [review]

**Comment by SamantaTarun** [comment]
@davide-baldo @CAOakleyII @polvorin please add me as a member of your organization. I want to contribute more in the future.

**Pull request merged**